Packs/SafeBreach/Integrations/SafeBreach/README.md

```diff
@@ -0,0 +1,1276 @@
+1. User get, create, update and delete.
+2. Deployment create, update and delete.
+3. Tests get and delete.
+4. Simulators get, update, delete.
+5. Get current tests/simulation status and/or queue them.
```

Comment:
what are these steps?

Reply:
Integration description.

Reply:
@merit-maita I have updated the integration description.